scripts/install-multi-user.sh

```diff
@@ -69,16 +69,17 @@ readonly PROXY_ENVIRONMENT_VARIABLES=(
     NO_PROXY
 )
 
-SUDO_EXTRA_ENVIRONMENT_VARIABLES=()
+SUDO_KEPT_ENVIRONMENT_VARIABLES=""
 
 setup_sudo_extra_environment_variables() {
-    local i=${#SUDO_EXTRA_ENVIRONMENT_VARIABLES[@]}
     for variable in "${PROXY_ENVIRONMENT_VARIABLES[@]}"; do
         if [ "x${!variable:-}" != "x" ]; then
-            SUDO_EXTRA_ENVIRONMENT_VARIABLES[i]="$variable=${!variable}"
-            i=$((i + 1))
+            SUDO_KEPT_ENVIRONMENT_VARIABLES="$SUDO_KEPT_ENVIRONMENT_VARIABLES,$variable"
         fi
     done
+
+    # Required by the darwin installer
+    export SUDO_KEPT_ENVIRONMENT_VARIABLES
```

Comment:
Why does it need to be exported, if all we do with it is pass it through `sudo`?

Reply:
Frankly: I don't know. But it failed complaining that the variable was unset before. So I assume we're executing something in a sub-script somewhere